importer/importer_utils.py

```diff
@@ -434,3 +434,16 @@ def create_wlm_url(country, language, id):
     url_base = "https://tools.wmflabs.org/heritage/api/api.php?action=search&format=json&srcountry={}&srlanguage={}&srid={}"
     return url_base.format(
         country, language, quote(id))
+
+
+def get_P31(q_number):
+    results = []
+    site = pywikibot.Site("wikidata", "wikidata")
+    repo = site.data_repository()
```

Comment:
Don't need this one. You can use `site` in `ItemPage` (since it is a Wikibase site). Ideally you would want to reuse a

Ideally you do not want to create a new site instance each time. Instead you want to create one early on (ideally in importer) and then reuse that in uploader and elsewhere. But I would break that out as a separate task.